lms/static/scripts/frontend_apps/components/test/ContentSelector-test.js

```diff
@@ -62,13 +63,15 @@ describe('ContentSelector', () => {
   const renderContentSelector = (props = {}) => {
     const noop = () => {};
     return mount(
-      <Config.Provider value={fakeConfig}>
-        <ContentSelector
-          setErrorInfo={noop}
-          onSelectContent={noop}
-          {...props}
-        />
-      </Config.Provider>
+      <IntlProvider messages={{}} locale="en" defaultLocale="en">
```

Comment:
As I noted in the PR description, `messages` would eventually hold translated messages for various locales, which we'd import from JSON data. There are examples in [`format-js` documentation](https://formatjs.io/docs/getting-started/installation) and in various tutorials around the web (I can provide some links if helpful).